### ISSUE TITLE
[Bug] GIF Recording Encoding Stuck and Locks Output File

### ISSUE THREAD

**Issue by angelhdz** [ShareX/ShareX#3974]
Hello guys! 

Wonderful tool. I replaced my old screen capture tool (a good one) because this was better.

I have an issue, since many versions ago. I have the latest one currently,   **v12.4.1**

When recording GIF, the **Encoding...** icon gets stuck, and I can't open the output file. 

By coincidence, I found out that by clicking that icon, it goes away and frees the file and I can then open it. I hope this get fixed soon. Thank you!

Your work is highly appreciated,

Angel

**Comment by angelhdz** [comment]
### Update
It seems like as there is no progress of the encoding process, the user couldn't know when the file is ready for usage, in other words, when the encoding process finishes.

But, anyways, there are times when the encoding finished few minutes ago, and the encoding icon is still displayed in the notification area of Window's toolbar, and it is locking the file, the file can not be used, unless we click the encoding icon and it goes away and frees the file.

**Comment by WiliTest** [comment]
ShareX 15.0 I recorded a video (last 3 hours), at the end I clicked on the red button to stop it. It doesn't do anything. Right clicking on it shows the stop abort + abort button.
![image](https://user-images.githubusercontent.com/24194840/228601241-313c62c0-af76-4880-ae80-35e9d80132cb.png)
 Stop doesn't stop it. the red icon is stuck
![image](https://user-images.githubusercontent.com/24194840/228603058-279c3b0f-0c0a-4c89-a891-c189c85eac2a.png)


In the screenshots folder, I can see a video file. Mousing over it shows first it was empty (size = 0), few minutes afer its size changed to 7,75MB, and remains like this for several minutes. I can't open it. What's going on?
![image](https://user-images.githubusercontent.com/24194840/228602919-66b0a836-2471-4315-9323-4116a2078f6d.png)
